packages/@okta/vuepress-theme-prose/components/PrimaryNav.vue

```diff
@@ -1,50 +0,0 @@
-<template>
```

Comment:
Is this component not being used by the "new" nav?

Reply:
I couldn't find any usages of the component in the project, it's all just in `Header.vue`